subjects/car_rental/README.md

```diff
@@ -0,0 +1,89 @@
+## car_rental
+
+### Instructions
+
+Create a struct named `Car` and another one named `RentalBusiness`.
+
+The scope of the exercise will be to modify and operate on the `Car` in the `RentalBusiness` even if this element is not declared as mutable.
+
```

Comment:
I would add some description of the aspected behaviour of the `RentalBusiness` methods that need to be implemented

Reply:
done